Example/Source/Examples/Interactive/InsertBrickViewController.swift

```diff
@@ -67,9 +67,14 @@ class InsertBrickViewController: BrickApp.BaseBrickController {
         updateRepeatCounts()
     }
 
-    func updateRepeatCounts() {
+    func removeBrick(indexPath: NSIndexPath) {
+        self.numberOfLabels -= 1
+        updateRepeatCounts([indexPath])
+    }
+
+    func updateRepeatCounts(fixedDeletedIndexPaths: [NSIndexPath]? = nil) {
         UIView.animateWithDuration(0.5, animations: {
-            self.brickCollectionView.invalidateRepeatCounts { (completed, insertedIndexPaths, deletedIndexPaths) in
+            self.brickCollectionView.invalidateRepeatCounts(false) { (completed, insertedIndexPaths, deletedIndexPaths) in
```

Comment:
not sure what false means at the call site here. could this parameter be given an external label?